tests/pipelines/main.nf.test

```diff
@@ -33,8 +32,80 @@ nextflow_pipeline {
             def actual_calls = path("$launchDir/results/call/Called/results.text")
             def expected_calls = path("$baseDir/tests/data/called/expected_results.txt")
             assert actual_calls.text == expected_calls.text
+
+            // Check IRIDA Next JSON output
+            assert path("$launchDir/results/iridanext.output.json").json == path("$baseDir/tests/data/irida/test_iridanext.output.json").json
+
+            def iridanext_json = path("$launchDir/results/iridanext.output.json").json
+            def iridanext_samples = iridanext_json.files.samples
+            def iridanext_metadata = iridanext_json.metadata.samples
+
+            assert iridanext_metadata.sampleQ."address" == "1.1.3"
```

Comment:
Could you add an assert in this case to make sure that `sampleQ` is the only sample listed in the metadata?

Reply:
Sure 😄 : [7feaee8](https://github.com/phac-nml/gasnomenclature/pull/10/commits/7feaee8f6c38a61ca6ef8d4916b808aff9f4645e)